docs/source/modin/supported/general_supported.rst

```diff
@@ -18,7 +18,7 @@ Data manipulations
 | ``concat``                  | P                               | ``levels`` is not supported,     |                                                    |
 |                             |                                 | ``copy`` is ignored              |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``crosstab``                | N                               |                                  |                                                    |
+| ``crosstab``                | Y                               |                                  |                                                    |
```

Comment:
I think it should be `P` and inherit all the parameter limitations from `pivot_table`.

Reply:
In this case, the limitations from pivot_table do not apply, since we do not accept the unsupported arguments from pivot_table, and index must be specified, so margins is fully supported.